src/Components/design/Hero/Hero.jsx

```diff
@@ -18,20 +19,29 @@ export default function Hero(props) {
         <div className="hero__header">
           <h1>{props.headerText}</h1>
           <div className="hero__buttons">
+            {!props.overView && (
```

Comment:
we might want to allow the hero component to take in the jsx instead now that we need custom control

something like
```jsx
<Hero
  side={
    <>
      <Button
        variant="outlined"
        as={CurrentOrganizationLink}
        to={props.copyLink}
      >
        Copy
      </Button>
      <Button
        variant="outlined"
        as={CurrentOrganizationLink}
        to={props.editLink}
      >
          Edit
        </Button>
    </>
  }
/>
```

Reply:
This also makes the hero component more reusable....